lib/Data/Xls.hs

```diff
@@ -104,6 +110,28 @@ decodeXls file =
             count <- liftIO $ c_xls_wb_sheetcount pWB
             mapM_ (decodeOneWorkSheet file pWB) [0 .. count - 1]
 
+
+-- | Parse a Microsoft excel xls workbook file into a [[[String]]].
+-- Nesting levels means worksheets rows and cells accordingly.
+--
+-- Throws 'XlsException'
+--
+decodeXlsIO
+  :: FilePath
+  -> IO [[[String]]]
+decodeXlsIO file = do
```

Comment:
Let's keep the indentation consistent as 4 spaces everywhere.

Reply:
Done.